regulations/static/components/RelatedRules.vue

```diff
@@ -31,12 +31,15 @@ export default {
     async created() {
         this.rules = await this.fetch_rules(this.title, this.part);
     },
-    
+
     methods: {
         async fetch_rules(title, part) {
-            const response = await fetch(`https://www.federalregister.gov/api/v1/documents.json?fields[]=type&fields[]=abstract&fields[]=citation&fields[]=correction_of&fields[]=dates&fields[]=docket_id&fields[]=docket_ids&fields[]=document_number&fields[]=effective_on&fields[]=html_url&fields[]=publication_date&fields[]=regulation_id_number_info&fields[]=regulation_id_numbers&fields[]=title&per_page=20&conditions[type][]=RULE&conditions[cfr][title]=${title}&conditions[cfr][part]=${part}`);
+            const response = await fetch(`https://www.federalregister.gov/api/v1/documents.json?fields[]=type&fields[]=abstract&fields[]=citation&fields[]=correction_of&fields[]=dates&fields[]=docket_id&fields[]=docket_ids&fields[]=document_number&fields[]=effective_on&fields[]=html_url&fields[]=publication_date&fields[]=regulation_id_number_info&fields[]=regulation_id_numbers&fields[]=title&per_page=20&order=newest&conditions[type][]=RULE&conditions[cfr][title]=${title}&conditions[cfr][part]=${part}`);
             const rules = await response.json();
-            return rules.results;
+            const results = rules.results.sort((a, b) => {
```

Comment:
consider pulling this into a function like `by_effective_on` or `compare_effective_on` so it becomes `rules.results.sort(by_effective_on)`.